store/CSDK/main.cpp

```diff
@@ -99,21 +102,187 @@ bool readFromLocalWithoutProjection(JNIEnv *env) {
         printf("%s\t", carbonRow.getDecimal(10));
         printf("%f\t", carbonRow.getFloat(11));
         printf("\n");
+        env->DeleteLocalRef(row);
+        env->DeleteLocalRef(array1);
     }
 
     carbonReaderClass.close();
 }
 
+/**
+ * test next Row Performance
+ *
+ * @param env  jni env
+ * @return
+ */
+bool testNextRowPerformance(JNIEnv *env, char *path, int printNum, char *argv[], int argc) {
```

Comment:
Same comment, this is internal comparision. No need to add in example test case.

Reply:
I want to test for big data(more than 100 million rows), I change the test case name, is it ok?

Reply:
Example code must be independent of data. It should work for small data as well as bigdata.
Also as you know we cannot keep huge data testcase in automation, PR builder will take time. So we avoid huge data test case. It is DFX scenario.